client/components/post.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+export default class Post extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isFavorite: null
+    };
+    this.handleFavoriteClick = this.handleFavoriteClick.bind(this);
+  }
+
+  componentDidMount() {
+    const token = window.localStorage.getItem('lfz-final');
+    if (token) {
+      fetch(`/api/post/${this.props.postData.postId}`, {
+        headers: {
+          'Content-Type': 'application/json',
+          'x-access-token': token
+        }
+      })
+        .then(res => res.json())
+        .then(result => {
+          this.setState({ isFavorite: result[0].isFavorite });
+        });
+    }
+  }
```

Comment:
You should not need this component did mount method because the props postData should already say if it `isFavorite`